### ISSUE TITLE
Running solium in a browser

### ISSUE THREAD

**Issue by subramanianv** [duaraghav8/Solium#238]
Is there a browerized build for solium ?

**Comment by duaraghav8** [comment]
Currently there isn't, I'm working with @yann300 and will hopefully come up with one asap

**Comment by ToJen** [comment]
Agreed, my approach has been to use `solhint` which has less features. See this [example](https://github.com/ToJen/chain-stats/blob/master/frontend/src/components/Lint.js).
